perses/tests/test_samplers.py

```diff
@@ -68,6 +69,7 @@ def test_valence():
             f.description = "Testing MultiTargetDesign sampler with %s transfer free energy from vacuum -> %s" % (testsystem_name, environment)
             yield f
 
+@skipIf(os.environ.get("TRAVIS", None) == 'true', "Skip test samplers for now to prevent holding up Travis")
```

Comment:
These tests are already controllable through `.travis.yml`.  You can see on lines 81-82 that if the `TESTSYSTEMS` environment variable is set, only those tests are run.  We can just set `TESTSYSTEMS` to `ValenceSmallMoleculeLibraryTestSystem` in `.travis.yml` and omit any other tests for `perses.tests.test_samplers`.